src/widgets/notes/notes.test.tsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { cleanup, render, wait } from "@testing-library/react";
+import { BrowserRouter } from "react-router-dom";
+import Notes from "./notes.component";
+
+describe("<NotesComponent />", () => {
+  let wrapper: any;
+
+  afterEach(cleanup);
+
+  xit("renders without dying", async () => {
+    wrapper = render(
+      <BrowserRouter>
+        <Notes />
+      </BrowserRouter>
+    );
+    await wait(() => {
+      expect(wrapper).toBeTruthy();
+    });
```

Comment:
I think this assertion is useless. `render()` always returns a wrapper under every circumstance, right? Or is that not true? The main thing that this test is doing is just verifying that `render()` doesn't throw an error - we don't need this line at all I don't think

Reply:
Yeah, it doesn't do much. I'll be removing this test for now as indicated above.